variation-commons-mongodb/src/main/java/uk/ac/ebi/eva/commons/mongodb/entities/SampleMongo.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.commons.mongodb.entities;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+import uk.ac.ebi.eva.commons.mongodb.entities.subdocuments.SamplePhenotypeMongo;
+
+import java.util.Set;
+
+@Document(collection = "#{mongoCollectionsSamples}")
+public class SampleMongo {
+
+    public static final String SEX_FIELD = "sex";
+
+    public static final String FATHER_FIELD = "father";
+
+    public static final String MOTHER_FIELD = "mother";
+
+    public static final String PHENOTYPE_FIELDS = "phenotypes";
```

Comment:
Is "phenotypes" being used as the attribute name instead of "categories" as described in the PR comment?

Reply:
yes, that word popped up yesterday offline

Reply:
exactly